### PR TITLE
STY: reformatted with pre-commit

### PR DESCRIPTION
**Pull request by ghalym**
## Pre-merge checklist
- [ ] Code works interactively
- [ ] Code contains descriptive comments
- [ ] Test suite passes locally
- [ ] Libraries are set to fixed versions and not ``Always Newest``
- [ ] Code committed with ``pre-commit`` (alternatively ``pre-commit run --all-files``)

### CODE REVIEW

File: Arbiter/_Config/IO/Device 1 (EtherCAT).xti [hunk 1]
```diff
@@ -44,11 +44,15 @@
 		<Box File="Box 23 (CU1128).xti" Id="23">
 			<EtherCAT PortABoxInfo="#x01000016"/>
 		</Box>
-		<EtherCAT EnableVirtualSwitch="true" MaxSwitchPorts="7" MaxSwitchFrames="220">
-			<Redundancy Mode="2" AdapterReference="true" PreviousPort="33554461"/>
+		<Box File="plc-txi-lfe-motion.xti" Id="40">
```
Comment (ZLLentz) [inline Arbiter/_Config/IO/Device 1 (EtherCAT).xti:47]: Looks like you have functional changes in here mixed in with the style update

File: Arbiter/_Config/IO/Device 1 (EtherCAT)/Box 40 (EL6695-0002).xti
Comment (ZLLentz) [inline Arbiter/_Config/IO/Device 1 (EtherCAT)/Box 40 (EL6695-0002).xti:1]: Is this a full new terminal added?

**Review by ZLLentz — APPROVED** [review]
The style update looks good
There was other stuff added here too, was it all intentional?

File: Arbiter/_Config/IO/Device 1 (EtherCAT)/Box 40 (EL6695-0002).xti
Comment (ghalym) [reply Arbiter/_Config/IO/Device 1 (EtherCAT)/Box 40 (EL6695-0002).xti:1]: Not a new terminal. We recently connected the TXI PLCs. So I scanned them and renamed them so they are ready for when we integrate them.

**Pull request merged**